src/components/InfoBox.vue

```diff
@@ -14,7 +14,7 @@ export default {
   data () {
     return {}
   },
-  props: ['color', 'action', 'icon'],
+  props: ['color', 'action', 'icon', 'title'],
```

Comment:
What do you think to transform this props in other properties of the Vuex global state?